### PR TITLE
Related applications review with proposals

### PR DESCRIPTION
**Pull request by anssiko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -308,7 +308,7 @@
         <li>Otherwise, return <code>false</code>.
         </li>
       </ol>
-      <div class="issue" title="🐒 Monkey patch">
+      <div class="issue" title="&#128018; Monkey patch">
```
Comment (marcoscaceres) [inline index.html:311]: @anssiko, this change is unrelated to the feature and the issue being addressed. Do you mind reverting this? We can do it as a separate fix

File: index.html [hunk 2]
```diff
@@ -1631,16 +1631,17 @@ <h3 id="applying">
           <code title="">related_applications</code> member
         </h3>
         <p>
+          A <dfn>related application</dfn> is an application provided by the
+          underlying application platform that has a relationship with the
```
Comment (marcoscaceres) [inline index.html:1635]: I'm not a fan of  "provided by" (because it might not be installed) - maybe "accessible to"?

File: index.html [hunk 3]
```diff
@@ -1631,16 +1631,17 @@ <h3 id="applying">
...
-        <p>
-          The <code>related_applications</code> member is as an indication of a
-          relationship of the web application and the listed applications. This
-          relationship is unidirectional and unless a listed application claims
-          the same relationship, the <a>user agent</a> MUST NOT assume a
-          bi-directional endorsement.
+          member lists <a>related application</a>s and serves as an indication
+          of such a relationship between web application and <a>related
+          application</a>s. This relationship is unidirectional and unless a
+          listed application claims the same relationship, the <a>user agent</a>
```
Comment (marcoscaceres) [inline index.html]: Out of curiosity, how does a listed app claim the same relationship?

File: index.html [hunk 4]
```diff
@@ -1717,19 +1718,14 @@ <h3 id="applying">
...
-          <code>true</code>, the user agent might want to suggest installing
-          one of the listed related applications instead.
+          member is a boolean value that is used as a hint for the user agent
+          to say that <a>related application</a>s should be preferred over the
+          web application. The user agent MUST consider the missing value as
+          equivalent to have it set to <code>false</code>. If the
+          <code>prefer_related_applications</code> is set to <code>true</code>,
+          and the user agent wants to suggest to install the web application,
+          the user agent might want to suggest installing one of the
+          <a>related application</a>s instead.
```
Comment (marcoscaceres) [inline index.html]: Change: 
`<a>related application</a>s`
To: 

```
<a title="related application">related applications</a>
```

File: index.html (hunk 1 above)
Comment (anssiko) [inline index.html:311]: My editor did not co-operate, so I was forced to convert it to an HTML entity on the same pass. Sorry.

File: index.html (hunk 2 above)
Comment (anssiko) [inline index.html:1635]: Sounds good. Fixed.

File: index.html (hunk 3 above)
Comment (anssiko) [inline index.html]: Also interested, although out of scope for the spec. Feel like suggesting a new feature to http://developer.android.com/guide/topics/manifest/manifest-intro.html and friends ?-)

File: index.html (hunk 4 above)
Comment (anssiko) [inline index.html]: Thanks for the ReSpec protip, fixed here and elsewhere.

**Comment by mounirlamouri** [comment]
Thank you for your comments Anssi. They look good to me.
